ParquetClassLibrary/Crafting/CraftingRecipe.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Collections.Generic;
+using ParquetClassLibrary.Sandbox.ID;
+
+namespace ParquetClassLibrary.Crafting
+{
+    /// <summary>
+    /// Models the ingredients and process needed to produce a new item.
+    /// </summary>
+    public struct CraftingRecipe
+    {
+        #region Defaults
```

Comment:
Are these defaults (the maximum widths) hard rules that can't be broken? They look like they're used for enforcing things in the constructor, so maybe this could be called something other than Defaults.

Reply:
Oops, yes, that's an error on my part.